Lesson9/Gokhan/styles.css

```diff
@@ -0,0 +1,104 @@
+
+/* i added text content, pre element and content centeren on page */
+/* when i was write and save */
+/* i can^t use live server for this work */
```

Comment:
I cannot say why it does not work for you exactly. When I right click on the index.html file I can certainly open it with liveserver and it works as usual. Could it be that you tried opening styles.css file by a mistake?

Also, i noticed that the name of the styles file was not the same as the name of the file in the link tag in HTML. Added suggestion for it in HTML. Could that be connected to your problem?

Reply:
it is working Anna, thanks for all